autogen/sm_callback_init.m

```diff
@@ -0,0 +1,61 @@
+% sm_callback_init
+%
+% This scripts initializes the model variables and parameters. The script 
+% runs before the simulation starts.  This is called in the InitFcn callback 
+% found in Model Explorer.
+%
+% Copyright 2019 Flat Earth Inc
+%
+% THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR IMPLIED,
+% INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY, FITNESS FOR A PARTICULAR
+% PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE
+% FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE,
+% ARISING FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
+%
+% Ross K. Snider
+% Flat Earth Inc
+% 985 Technology Blvd
+% Bozeman, MT 59718
+% support@flatearthinc.com
+%% Create the control signals
+%mp = sm_init_control_signals(mp);  % create the control signals
+
+%% Create test signals for the left and right channels
+
+[modelPath,modelAbbreviation,~] = fileparts(which(bdroot));
+mp.modelPath = char(modelPath);
+mp.modelAbbreviation = char(modelAbbreviation);
+onPath = contains(path, [modelPath, pathsep]);
+if ~onPath
+    addpath(modelPath)
+end
```

Comment:
This just determines if the model directory is on the path, right? The section head comment above is misleading and should be moved below this block.